Projections/WinUI/WinUI.csproj

```diff
@@ -12,9 +12,9 @@
     <NoWarn>8305;0618</NoWarn>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
-    <DebugType>full</DebugType>
-    <DebugSymbols>true</DebugSymbols>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
+    <DebugType>full</DebugType>
```

Comment:
Is there a compelling reason to use Full PDBs rather than Portable?

Reply:
This change was done by VS, not by myself. I don't know of any particular reason to use full PDBs.

Reply:
I would highly recommend portable PDBs if possible.